checkov/main.py

```diff
@@ -401,6 +402,8 @@ def add_parser_args(parser):
     parser.add('--skip-cve-package',
                help='filter scan to run on all packages but a specific package identifier (denylist), You can '
                     'specify this argument multiple times to skip multiple packages', action='append', default=None)
+    parser.add('--prisma_api_url',
```

Comment:
I think this should go up next to the API key option.

Reply:
Done.